_dbsetup.py

```diff
@@ -0,0 +1,87 @@
+from json import load, dump
+from typing import Dict, List, Set
+from decimal import Decimal
+from enum import Enum
+from types import GeneratorType
+from pathlib import Path
+from random import shuffle
+import datetime
+
+import boto3
+from boardgamegeek import BGGClient, CacheBackendSqlite
+
+from app.schema import Game
+from app.utils import oxford_join, UniversalEncoder
+
+delete_existing = input("Delete existing table? y/N\n") or "N"
+if (delete_existing[0] in ["y", "Y"]):
+    delete_existing = True
+else:
+    delete_existing = False
+
+
+def create_table():
```

Comment:
I would consider making this file non-interactive and turning it into a pytest fixture, so that you can effectively include an initialized database by referring to the fixture's name in your test's list of arguments.

Reply:
Also, I'd consider putting these fixtures in tests/conftest.py

Reply:
thanks for all your feedback, @kurtiss !

I have a question about the `_dbsetup.py` -> `pytest fixture` comment. 

I ended up writing `_dbsetup.py` inorder to reinstantiate my actual dynamodb table, since I changed the naming of my table schema to be more pythonic, so this wasn't intended for testing purposes. I already have a [setup_db](https://github.com/moui72/bgt/blob/fbc01a4489c7532daa6427e7bb3b778e7e4a7a9f/tests/test_bgt.py#L48) fixture (currently in test_bgt.py; will move to conftest.py) which only inserts 25 games into the db. this allows me to set one game aside to do:

```
def test_get_nogame(outside_game_data, setup_db):
  test_game = outside_game_data
  response = client.get(f"/games/{test_game.id}")
    assert response.status_code == 404
```

which both tests my 404 response for that endpoint and that the db I'm querying is the mocked one and not the actual db.

so my question in: do you foresee a need for my current `setup_db` fixture, or should the code in `_dbsetup.py` replace it? would I ever want both?

is something like `_dbsetup.py`  in its original purpose something that needn't/shouldn't be included in the repo? i thought it might be helpful if someone ever forked the repo to see how they can use boto3 to create the appropriate table

Reply:
PS I think you're suggesting that all my fixtures should go into `tests/conftest.py` -- let me know if that's not what you meant